src/main/java/ru/turlyunef/Merge/MergeSortHandler.java

```diff
@@ -14,11 +14,26 @@
     Sort mergeSorter = new Sort();
     Parameters parameters;
 
-    public MergeSortHandler(Parameters parameters){
+    public MergeSortHandler(Parameters parameters) {
         this.parameters = parameters;
         mergeSorter.parameters = parameters;
```

Comment:
I think you shouldn't use this assignment, use method - setter() or pattern Builder